packages/react-network/src/context.tsx

```diff
@@ -0,0 +1,21 @@
+// https://gist.github.com/tyscorp/24516f1fdb8974a637b075cc9167a690
+
+import * as React from 'react';
+import {Manager, NoopManager} from './manager';
+
+const {Consumer, Provider} = React.createContext<Manager>(new NoopManager());
```

Comment:
Why do we initialize the default value in the context when it gets overridden when the Provider is created below? Shouldn't line 13 be `{manager = new NoopManager(), children}: Props` or am I missing something?

Reply:
It's kind of a matter of preference. This just makes it so that, if anyone uses the consumer directly with no provider in the tree, they don't get an `undefined`, they get a `Manager` (it just doesn't do anything)